internal/service/keyspaces/keyspace_test.go

```diff
@@ -70,19 +70,19 @@ func TestAccKeyspacesKeyspace_replicationSpecificationMulti(t *testing.T) {
 				Config: testAccKeyspaceConfig_replicationSpecification(rName, string(types.RsSingleRegion)),
 				Check: resource.ComposeAggregateTestCheckFunc(
 					testAccCheckKeyspaceExists(ctx, resourceName),
-					acctest.CheckResourceAttrRegionalARN(resourceName, "arn", "cassandra", "/keyspace/"+rName+"/"),
-					resource.TestCheckResourceAttr(resourceName, "name", rName),
-					resource.TestCheckResourceAttr(resourceName, "tags.%", "0"),
+					acctest.CheckResourceAttrRegionalARN(resourceName, "arn", "cassandra", "/keyspace/"+acctest.CtName+"/"),
```

Comment:
not sure if this should be made in this PR but the semgrep checks were failing due to these https://github.com/hashicorp/terraform-provider-aws/actions/runs/11596905595/job/32289156483